content/actions/creating-actions/creating-a-docker-container-action.md

```diff
@@ -200,11 +202,11 @@ jobs:
 
 #### Example using a private action
 
-Copy the following example workflow code into a `.github/workflows/main.yml` file in your action's repository. You can also replace the `who-to-greet` input with your name.
+Copy the following example workflow code into a `.github/workflows/main.yml` file in your action's repository. You can also replace the `who-to-greet` input with your name. This private action can't be published to {% data variables.product.prodname_marketplace %}, and can only be used in this repository.
```

Comment:
Like the public action comment, we should try to version publishing to GitHub Marketplace as only for `free-pro-team@latest` (it's not applicable to Enterprise Server).